### PR TITLE
Tdr 666 read notify params permissions

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl [hunk 1]
```diff
@@ -17,7 +17,9 @@
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/backend_checks_client/secret",
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/realm_admin_client/secret",
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/configuration_properties",
-        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/user_admin_client/secret"
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/user_admin_client/secret",
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/govuk_notify/api_key",
```
Comment (suzannehamilton) [inline modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl:21]: Just to check - why does Jenkins need permission to read the Notify API key? Does it copy the key to each environment?

Comment (TomJKing) [reply modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl:21]: This gets attached to the terraform iam role to give it permission to create the ssm params in each of the environments

Comment (suzannehamilton) [reply modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl:21]: Oh I see - this creates the parameter within SSM, but you have to set it manually in each account?

**Review by suzannehamilton — APPROVED** [review]

File: modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl (hunk 1 above)
Comment (TomJKing) [reply modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl:21]: Correct. Sorry should have made that clear.

**Pull request merged**